app.js

```diff
@@ -34,6 +34,7 @@ app.use(helmet({
 }));
 app.use(bodyParser.json());
 app.use('/client', express.static(path.join(__dirname, 'dist')));
+app.get('/Coral.js', (req, res) => res.sendFile(path.join(__dirname, 'client/coral-embed-api/index.js')));
```

Comment:
I defer to you on exact best URL, though I personally prefer the aesthetics of this.

I definitely recommend using a URL that doesn't depend on the structure of `dist/` or `client/` (as the current /client/ route does), as those details will change over time, and you don't want embed codes to have to change.